### ISSUE TITLE
Saving a sticker with Save Image As produces 2 save dialogs

### ISSUE THREAD

**Issue by Defman21** [telegramdesktop/tdesktop#1805]
### Steps to reproduce
1. Ask someone to send you a sticker.
2. Open context menu for it, press Save Image As.
3. Press Esc to abort the saving process.

### Expected behaviour
The save window closes

### Actual behaviour
The save window closes and immediately opens again, pressing Esc again closes it

### Configuration
**Operating system:** Ubuntu 15.10

**Version of Telegram Desktop:** 0.9.34dev

### Logs
```
[16:11:30.991 00-0000485] Shortcut override declined: 16777216 (window.cpp : 1045)
[16:11:30.991 00-0000486] Shortcut override declined: 16777216 (window.cpp : 1045)
[16:11:31.367 00-0000487] Shortcut override declined: 16777216 (window.cpp : 1045)
[16:11:31.367 00-0000488] Shortcut override declined: 16777216 (window.cpp : 1045)
```

**Comment by DaFri-Nochiterov** [comment]
_I thought to post it yesterday :laughing:_

Same on Ubuntu 15.10 Cinnamon DE

**Comment by auchri** [comment]
can't reproduce in 0.9.32

**Comment by DaFri-Nochiterov** [comment]
May be it's Linux-only problem?

**Comment by auchri** [comment]
Or an issue of 0.9.34dev

**Comment by diazbastian** [comment]
@auchri yeah, is a 0.9.34 dev issue (fedora 23 64 bit/windows 10). I can't reproduce in TDesktop 0.9.32

**Comment by stek29** [comment]
It opens two windows for GIFs too
0.9.34 dev GitHub OS X

**Comment by stek29** [comment]
Also opens 2 windows for: documents, videos, audio files and voice messages
So problem is in `history widget.cpp`'s `HistoryInner::saveContextFile()`

**Comment by stek29** [comment]
Oh, it's already fixed in [96e1f8e7cd1c824e68b992ffb7dbf9f488aa5578](https://github.com/telegramdesktop/tdesktop/commit/96e1f8e7cd1c824e68b992ffb7dbf9f488aa5578)

**Comment by Defman21** [comment]
0.9.36 - still here.

**Comment by stek29** [comment]
I mean sources are already fixed, so my research is unnecessary.
You could pull latest dev branch and build it, or wait for update.

**Comment by john-preston** [comment]
@Defman21 What about 0.9.39 dev? Still bad?

**Comment by Defman21** [comment]
@john-preston nope, it's fixed in 0.9.38dev for me.
